cli/asc.js

```diff
@@ -347,7 +410,7 @@ exports.main = function main(argv, options, callback) {
   const transforms = [];
   if (args.transform) {
     let tsNodeRegistered = false;
-    let transformArgs = args.transform;
+    let transformArgs = args.transform.map(resolve).filter(unique());
```

Comment:
It seems `unique` used once in this manner. How about refactor this to:
```js
function unique(array) {
  return [...new Set(array)];
}
...

let transformArgs = unique(args.transform.map(resolve));
```

Reply:
Definitely. I agree with this.

```ts
return Array.from(new Set(array));
```

Would this be better, or does using the spread syntax fit the code style better?

Reply:
Spread syntax just shorter. but it not big deal

Reply:
I went with @MaxGraey's suggestion because it's smaller.